### DISCUSSION TITLE
Battery aggregator shows 0% SOC sometimes even if the batterys have about 20-30% SOC

### DISCUSSION THREAD

**Discussion by Martinmaker92** [Louisvdw/dbus-serialbattery#883] [Q&A]
I am using 3 Batttery Packs with 18S 230Ah JK BMS on my victron system. The battery aggregator works fine for most of the time, but sometimes it shows 0% soc and the MP try to lad the pack. Not sure what to do!

Can anybody help me?

![dsfad](https://github.com/Louisvdw/dbus-serialbattery/assets/148790318/e81c1edc-5186-4dcc-b8fd-52682ce5886b)
![battery aggregator](https://github.com/Louisvdw/dbus-serialbattery/assets/148790318/bbcd1901-789e-432f-890e-995682738a42)

**Comment by mr-manuel** [comment]
You should open an issue in the repository of the aggregator you are using.

**Comment by Martinmaker92** [comment]
Ok i will do it later
